xml/Sector.xml

```diff
@@ -81,7 +81,7 @@
             </description>
             <category>112</category>
         </codelist-item>
-        <codelist-item status="active">
+        <codelist-item status="withdrawn">
```

Comment:
@akmiller01  let's check this. There are a number of codes that have been changed  from active to withdrawn and this looks incorrect to me:
1)  Is there a  "withdrawn-date" that you can identify in the source XML?
2) This is a "voluntary code" in OECD DAC terms. In IATI we don't differentiate between voluntary and non-voluntary, so my thinking is there is some error there.  I think the status should still be "active".
![image](https://user-images.githubusercontent.com/24607542/76516431-563b2000-6453-11ea-9521-6c2e053bd289.png)
 Let's cross-check this for other codes.